graphistry/embed_utils.py

```diff
@@ -0,0 +1,117 @@
+import torch
+import torch.nn as nn
+from torch.utils.data import Dataset, DataLoader
```

Comment:
@silkspace  can you walk him through lazy imports?

Reply:
@lmeyerov Thanks for mentioning, I have seen it, will integrate it. Need to add many more things like, exception handling, asserts, unit tests. I will work on those once I set up the pipeline 😃